src/web/routes/application/send-code/notify.js

```diff
@@ -0,0 +1,32 @@
+const { NotifyClient } = require('notifications-node-client')
+const uuid = require('uuidv4')
+const config = require('../../../../config')
+const notifyClient = new NotifyClient(config.environment.NOTIFY_API_KEY)
+const { logger } = require('../../../logger')
+require('dotenv').config()
+
+const { EMAIL, TEXT } = require('../common/constants')
+
+const EMAIL_TEMPLATE_ID = '18cf4f69-e1b5-4aa9-bed7-c906d3a59285'
+const SMS_TEMPLATE_ID = 'a770479a-097c-4079-acc2-d452a6f21585'
```

Comment:
Where do these values come from, some documentation would be good here, how do I know if they're right or not when reviewing?

Reply:
Presumably these templates are specific to the 2FA process, so might be better to rename these constants so they are specific to that process.

Reply:
Added a comment stating that these match the templates in govuk, but won't rename as the context of send-code implies 2FA.